cli/command/context/import.go

```diff
@@ -0,0 +1,45 @@
+package context
+
+import (
+	"io"
+	"os"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/context/store"
+	"github.com/spf13/cobra"
+)
+
+func newImportCommand(dockerCli command.Cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "import [OPTIONS] CONTEXT FILE|-",
```

Comment:
Looks like it has no options;

```suggestion
		Use:   "import CONTEXT FILE|-",
```